doc/user_guide.md

```diff
@@ -386,7 +391,12 @@ Refer to spdm_server_init() in [spdm_responder.c](https://github.com/DMTF/spdm-e
    }
    ```
 
-   1.6, if PSK is required, optionally deploy PSK Hint.
+   1.6, if public key of responder was provisoned for requester, the provisioned_slot_id is which slot_id cert_chain was provsioned for requester.
+   ```
+   libspdm_set_data (spdm_context, LIBSPDM_DATA_LOCAL_PUBLIC_CERT_CHAIN_DEFAULT_SLOT_ID, NULL, provisioned_slot_id, sizeof(provisioned_slot_id));
+   ```
+
+   1.7, if PSK is required, optionally deploy PSK Hint.

```

Comment:
This text is confusing. I suggest no update, so far.

We have another big patch to support raw public key provision. Will discuss later.

Please drop the user_guide update in this patch.

Reply:
Thanks. I have deleted the user_guide change.